Gemfile

```diff
@@ -47,7 +47,7 @@ gem 'attr_encrypted', '3.1.0'
 gem 'aws-sdk-s3', '~> 1'
 gem 'aws-sdk-sns', '~> 1'
 gem 'betamocks', git: 'https://github.com/department-of-veterans-affairs/betamocks', branch: 'master'
-gem 'bgs_ext', git: 'https://github.com/department-of-veterans-affairs/bgs-ext.git', require: 'bgs'
+gem 'bgs_ext', git: 'https://github.com/department-of-veterans-affairs/bgs-ext.git', require: 'bgs', branch: 'slakrhakr/API-6454'
```

Comment:
This needs to switch back to master when [this pr](https://github.com/department-of-veterans-affairs/bgs-ext/pull/59) for bgs-ext gets merged.

Reply:
ok, that pr was merged and this has been updated